ozi.cc

```diff
@@ -267,7 +245,7 @@ ozi_openfile(const QString& fname)
    */
 
   if (fname == "-") {
-    if (ozi_file.file == nullptr) {
+    if (stream == nullptr) {
```

Comment:
yes, it can be depending on the number of waypoints, tracks and routes.

when fname == "-" then the first time here stream will be nullptr, but subsequent calls it will be != nullptr.

The first call possibility is from data_write() if we have one or more waypoints.
Subsequent call possibilities are from data_write() if we have tracks, one call per track header.
And a final possibility from data_write() if we have one or more routes.